src/Microsoft.Azure.SignalR.Common/Endpoints/IConfigurationExtension.cs

```diff
@@ -0,0 +1,32 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Linq;
+using Microsoft.Extensions.Configuration;
+
+namespace Microsoft.Azure.SignalR.Common.Endpoints
+{
+    public static class IConfigurationExtension
```

Comment:
internal

Reply:
This will used by function extension?

Reply:
`[EditorBrowsable(EditorBrowsableState.Never)]` added.

Reply:
is it possible to internal visible to the function extension instead?

Reply:
Finished.